src/__tests__/home.test.js

```diff
@@ -0,0 +1,14 @@
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import store from '../redux/configureStore';
+import Home from '../components/Home';
+
+test('Home renders correctly', () => {
+  const tree = renderer.create(
+    <Provider store={store}>
+      <Home />
+    </Provider>,
+  )
+    .toJSON();
```

Comment:
- [ ] In the requirement, you must have [unit tests](https://felixgerschau.com/unit-testing-react-introduction/) and [integration tests](https://www.toptal.com/react/react-testing-library-tutorial#:~:text=Integration%20tests%20are%20a%20sweet,maintainability%20without%20impairing%20development%20speed.). But I only can see that there are just snapshot tests. Kindly add those tests as well. Requirement highlight for [writing tests](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Testing%20requirements,the%20Redux%20Store.) for the project.